include/delivery.php

```diff
@@ -30,30 +30,6 @@ function delivery_run(&$argv, &$argc){
 
 		$contact_id = intval($argv[$x]);
 
-		/// @todo When switching completely to the worker we won't need this anymore
```

Comment:
Yeah, I thought of this after I submitted the original PR. Since it wasn't accepted I added it in this PR.